src/pages/using-spark/guides/content-style-guidelines.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Content Style Guidelines
+---
+
+# Content Style Guidelines
+
+UX Writers, designers and developers can use the **Content Style Guidelines** as a guide to writing style, usage, text for copy, documentation, reference information, and copy for all Rocket Products digital experiences.
+
+### Table Of Contents
+
+- [Capitalization](#capitalization)
+  - [Sentence Case](#sentence-case)
+  - [Start Case](#start-case)
+- [Quotation Marks](#quotation-marks)
+- [Accessibility Guidelines](#accessibility-guidelines)
```

Comment:
Refer to how we did this in color usage page. For some reason markdown converts these nested lists and puts unecessary spacing between them: 
![image](https://user-images.githubusercontent.com/4342363/121391437-5eb32c80-c91c-11eb-9197-f88a699983c4.png)


https://raw.githubusercontent.com/sparkdesignsystem/spark-design-system/staging/src/pages/using-spark/foundations/color-usage.mdx

Reply:
Thank you, I added the class to indent the list items in the '<ul>' and it worked!